docs/Catalog/SearchQuery.md

```diff
@@ -23,13 +23,13 @@ There are two types of indexing in Quilt:
 * *shallow* indexing includes object metadata (such as the file name and size)
 * *deep* indexing includes object contents. Quilt supports deep
```

Comment:
Do we note (if not we should), that:
* we only index a limited number of bytes for specified file formats
* both the maximum byte extract per document and file formats to deep index are controlled in the Catalog's Admin Panel